packages/app/components/pack_table/TableItem.tsx

```diff
@@ -58,34 +66,27 @@ const TableItem = ({
       name,
       `${formatNumber(weight)} ${unit}`,
       quantity,
-      <PackOptions
-        Edit={
-          <EditPackItemModal>
-            <AddItem
-              _id={_id}
-              packId={_id}
-              isEdit={true}
-              initialData={itemData}
-            />
-          </EditPackItemModal>
-        }
-        Delete={
-          <DeletePackItemModal
-            itemId={_id}
-            pack={currentPack}
-            refetch={refetch}
-            setRefetch={setRefetch}
-          />
-        }
-        Ignore={
-          <IgnoreItemCheckbox
-            itemId={_id}
-            isChecked={checkedItems.includes(_id)}
-            handleCheckboxChange={handleCheckboxChange}
-          />
-        }
-        editTrigger={onTrigger}
-      />,
+      <DropdownMenu.Root>
```

Comment:
let's just put this in a component in this same file for easier readability ie ItemDropdown or something